app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
why different .gitignore files ?


merge this one with the root .gitignore

add /app/build there itself